.github/workflows/rust.yml

```diff
@@ -0,0 +1,36 @@
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types:
+
```

Comment:
what does that do?

Reply:
That being `types:`? Nothing much really, it continues using the default set of pull request events as a trigger to invoke the action.